src/js_realm.hpp

```diff
@@ -1582,27 +1590,27 @@ void RealmClass<T>::writeCopyTo(ContextType ctx, ObjectType this_object, Argumen
         config = write_copy_to_helper_deprecated(ctx, this_object, args);
     }
 
-    bool realm_is_synced = static_cast<bool>(realm->sync_session());
-    bool copy_is_synced = static_cast<bool>(config.sync_config);
-
-    if (realm_is_synced && !copy_is_synced) {
-        // case 3)
-        Group& group = realm->read_group();
-        group.write(config.path, config.encryption_key.empty() ? nullptr : config.encryption_key.data());
-        return;
-    }
-    else if (realm_is_synced && copy_is_synced) {
-        // case 4)
-        BinaryData binary_encryption_key;
-        if (!config.encryption_key.empty()) {
-            binary_encryption_key = std::move(BinaryData(config.encryption_key.data(), config.encryption_key.size()));
-        }
-        realm->write_copy(config.path, binary_encryption_key);
-        return;
-    }
-
-    // case 1), 2)
-    realm->export_to(config);
+    // bool realm_is_synced = static_cast<bool>(realm->sync_session());
+    // bool copy_is_synced = static_cast<bool>(config.sync_config);
+
+    // if (realm_is_synced && !copy_is_synced) {
+    //     // case 3)
+    //     Group& group = realm->read_group();
+    //     group.write(config.path, config.encryption_key.empty() ? nullptr : config.encryption_key.data());
+    //     return;
+    // }
+    // else if (realm_is_synced && copy_is_synced) {
+    //     // case 4)
+    //     BinaryData binary_encryption_key;
+    //     if (!config.encryption_key.empty()) {
+    //         binary_encryption_key = std::move(BinaryData(config.encryption_key.data(), config.encryption_key.size()));
+    //     }
+    //     realm->convert(config.path, binary_encryption_key);
+    //     return;
+    // }
+
+    // // case 1), 2)
```

Comment:
Can this commented out code be removed?

Reply:
Yes, the code can be removed (already done on `master`)